### PR TITLE
Add "Streams" section.

### PR DESCRIPTION
**Pull request by yutakahirano**
Add "Streams" section as part of the effort to add the body property to Response (#118).

### CODE REVIEW

**Comment by yutakahirano** [comment]
Hmm, I just noticed that Overview.src.html was roughly written in 80-columns. I will fix the change.

**Comment by annevk** [comment]
It's probably okay to migrate to 100 columns.

**Comment by annevk** [comment]
Standardizing on 100 columns across WHATWG is probably easiest at this point.

**Comment by yutakahirano** [comment]
Done. Can you take a look?

File: Overview.src.html [hunk 1]
```diff
@@ -1056,7 +1056,150 @@ <h4 id=terminology-headers>Headers</h4>
 <p class=note>The details of referrer policies are discussed in Referrer Policy.
 <span data-anolis-ref>REFERRER</span>
 
+<h3>Streams</h3>
 
+<h4>ReadableStream</h4>
```
Comment (annevk) [inline Overview.src.html]: Add newline.

File: Overview.src.html [hunk 2]
```diff
@@ -1056,7 +1056,150 @@ <h4 id=terminology-headers>Headers</h4>
...
+<p>A <dfn title=concept-readable-stream>ReadableStream</dfn> represents a stream of data as
+specified in <span data-anolis-ref>STREAMS</span>. In this section, we define common operations
```
Comment (annevk) [inline Overview.src.html]: Say `<cite>Streams</cite>` here, add the reference at the end of the paragraph.

File: Overview.src.html [hunk 3]
```diff
@@ -1056,7 +1056,150 @@ <h4 id=terminology-headers>Headers</h4>
...
+for ReadableStream used in this spec.</p>
```
Comment (annevk) [inline Overview.src.html]: Drop " used in this spec".

File: Overview.src.html [hunk 4]
```diff
@@ -1056,7 +1056,150 @@ <h4 id=terminology-headers>Headers</h4>
 <p class=note>The details of referrer policies are discussed in Referrer Policy.
 <span data-anolis-ref>REFERRER</span>
 
+<h3>Streams</h3>
 
+<h4>ReadableStream</h4>
+<p>A <dfn title=concept-readable-stream>ReadableStream</dfn> represents a stream of data as
```
Comment (annevk) [inline Overview.src.html]: Just make this `concept-ReadableStream`.

File: Overview.src.html [hunk 5]
```diff
@@ -1056,7 +1056,150 @@ <h4 id=terminology-headers>Headers</h4>
...
+specified in <span data-anolis-ref>STREAMS</span>. In this section, we define common operations
+for ReadableStream used in this spec.</p>
+
+<p>To <dfn title=concept-enqueue-readable-stream>enqueue</dfn> a byte sequence
+(<var title>data</var>) into a <span title=concept-readable-stream>ReadableStream</span>
+<var title>stream</var>, run the following step.
```
Comment (annevk) [inline Overview.src.html]: Use "run these steps:" (it's okay that it's just one for now).

File: Overview.src.html [hunk 6]
```diff
@@ -1056,7 +1056,150 @@ <h4 id=terminology-headers>Headers</h4>
...
+
+<ol>
+ <li><p>Call <code data-anolis-spec=streams>EnqueueInReadableStream</code> (<var title>stream</var>,
+        <var title>data</var>). Rethrow any exceptions.
```
Comment (annevk) [inline Overview.src.html]: Only indent as far as `<li>`.

File: Overview.src.html [hunk 7]
```diff
@@ -1056,7 +1056,150 @@ <h4 id=terminology-headers>Headers</h4>
...
+<ol>
+  <li><p>Call <code data-anolis-spec=streams>CloseReadableStream</code>
+         (<var title>stream</var>). Rethrow any exceptions.
+</ol>
+</p>
+
+<p>To <dfn title=concept-construct-readable-stream>construct</dfn> a
+<span title=concept-readable-stream>ReadableStream</span> with given <var title>strategy</var>,
+<var title>pull</var> action and <var title>cancel</var> action, all of which are optional, run
+these steps.
```
Comment (annevk) [inline Overview.src.html]: Use ":" at the end.

File: Overview.src.html [hunk 8]
```diff
@@ -1056,7 +1056,150 @@ <h4 id=terminology-headers>Headers</h4>
...
+<p class="note no-backref">Because the reader grants exclusive access, the actual mechanism of how
+to read cannot be observed. Implementations could use more direct mechanism if convenient.</p>
+
+<p>To <dfn title=concept-tee-readable-stream>tee</dfn> a
+<span title=concept-readable-stream>ReadableStream</span> <var title>stream</var>, run the following
+step.
+<ol>
+  <li><p>Return the result of calling
+         <code data-anolis-spec=streams>TeeReadableStream</code>(<var title>stream</var>, true).
+         Rethrow any exception.
```
Comment (annevk) [inline Overview.src.html]: Align indentation with `<li>`.

File: Overview.src.html [hunk 9]
```diff
@@ -1056,7 +1056,150 @@ <h4 id=terminology-headers>Headers</h4>
...
+  </ul>
+  <li><p>Return <var title>promise</var>.
+</ol></p>
+
+<p class="note no-backref">Because the reader grants exclusive access, the actual mechanism of how
+to read cannot be observed. Implementations could use more direct mechanism if convenient.</p>
+
+<p>To <dfn title=concept-tee-readable-stream>tee</dfn> a
+<span title=concept-readable-stream>ReadableStream</span> <var title>stream</var>, run the following
+step.
```
Comment (annevk) [inline Overview.src.html]: Use ":" and follow by a newline.

File: Overview.src.html [hunk 10]
```diff
@@ -2816,7 +2959,7 @@ <h3 id=http-cors-protocol>CORS protocol</h3>
 
 <p class=no-backref>The <code title=dom-global-fetch>fetch()</code> method is relatively
 low-level API for <span title=concept-fetch>fetching</span> resources. It covers slightly
-more ground than <code data-anolis-spec=xhr>XMLHttpRequest</code>, although it is
+more ground than <code data-anolis-spec=xhr>XMLHttpRequest</code>,grou although it is
```
Comment (annevk) [inline Overview.src.html]: This looks wrong.

**Comment by annevk** [comment]
Also, newline after each `<li>` is done.

File: Overview.html [hunk 11]
```diff
@@ -1111,7 +1113,150 @@ <h3 id="referrer-policies"><span class="secno">3.4 </span>Referrer policies</h3>
...
+</ol><p></p>
+
+<p>To <dfn id="concept-read-all-bytes-from-readable-stream" title="concept-read-all-bytes-from-readable-stream">read</dfn> all bytes from a
+<a href="#concept-readable-stream" title="concept-readable-stream">ReadableStream</a> with <var title="">reader</var>, run these
+steps.
+
+<ol>
+  <li><p>Let <var title="">promise</var> be a new promise.
+  <li><p>Let <var title="">bytes</var> be an empty byte sequence.
+  <li><p>Let <var title="">read</var> be the result of calling
```
Comment (annevk) [inline Overview.html]: Because of the nested `<ul>`, this `<p>` needs to start on its own line.

File: Overview.src.html (hunk 1 above)
Comment (yutakahirano) [inline Overview.src.html]: Done.

File: Overview.src.html (hunk 2 above)
Comment (yutakahirano) [inline Overview.src.html]: Done.

File: Overview.src.html (hunk 3 above)
Comment (yutakahirano) [inline Overview.src.html]: Done.

File: Overview.src.html (hunk 4 above)
Comment (yutakahirano) [inline Overview.src.html]: Done.

File: Overview.src.html (hunk 5 above)
Comment (yutakahirano) [inline Overview.src.html]: Done.

File: Overview.src.html (hunk 6 above)
Comment (yutakahirano) [inline Overview.src.html]: Done.

File: Overview.src.html (hunk 7 above)
Comment (yutakahirano) [inline Overview.src.html]: Done.

File: Overview.src.html (hunk 8 above)
Comment (yutakahirano) [inline Overview.src.html]: Done.

File: Overview.src.html (hunk 9 above)
Comment (yutakahirano) [inline Overview.src.html]: Done.

File: Overview.src.html [hunk 12]
```diff
@@ -1056,7 +1056,158 @@ <h4 id=terminology-headers>Headers</h4>
...
+<h3>Streams</h3>
 
+<h4>ReadableStream</h4>
+
+<p>A <dfn title=concept-ReadableStream>ReadableStream</dfn> represents a stream of data as
+specified in <cite>Streams</cite>. In this section, we define common operations for ReadableStream.
+<span data-anolis-ref>STREAMS</span>
+</p>
+
+<p>To <dfn title=concept-enqueue-readable-stream>enqueue</dfn> a byte sequence
```
Comment (annevk) [inline Overview.src.html]: So the convention would be `concept-ReadableStream-enqueue` and the same for the other defined terms.

File: Overview.src.html (hunk 10 above)
Comment (yutakahirano) [inline Overview.src.html]: Sorry, done.

File: Overview.src.html [hunk 13]
```diff
@@ -1056,7 +1056,158 @@ <h4 id=terminology-headers>Headers</h4>
...
+(<var title>data</var>) into a <span title=concept-ReadableStream>ReadableStream</span>
+<var title>stream</var>, run the these steps:
+
+<ol>
+ <li><p>Call <code data-anolis-spec=streams>EnqueueInReadableStream</code> (<var title>stream</var>,
+     <var title>data</var>). Rethrow any exceptions.
```
Comment (annevk) [inline Overview.src.html]: The `<var>` needs to be on the same line as the `<li>`.

File: Overview.html (hunk 11 above)
Comment (yutakahirano) [inline Overview.html]: Done.

File: Overview.src.html [hunk 14]
```diff
@@ -1056,7 +1056,158 @@ <h4 id=terminology-headers>Headers</h4>
...
+</ol>
+</p>
+
+<p>To <dfn title=concept-close-readable-stream>close</dfn> a
+<span title=concept-ReadableStream>ReadableStream</span> <var title>stream</var>, run these steps:
+<ol>
+  <li><p>Call <code data-anolis-spec=streams>CloseReadableStream</code>
```
Comment (annevk) [inline Overview.src.html]: Only a single space is used elsewhere.

File: Overview.src.html (hunk 13 above)
Comment (yutakahirano) [inline Overview.src.html]: Can you tell me why?

Comment (annevk) [inline Overview.src.html]: That's the convention used throughout this document.

File: Overview.src.html (hunk 12 above)
Comment (yutakahirano) [inline Overview.src.html]: Sorry that my commenting timing was confusing. Fixed.

File: Overview.src.html (hunk 13 above)
Comment (yutakahirano) [inline Overview.src.html]: Ah, I see. I misunderstood your previous comment "Only indent as far as <li>". Thanks.

File: Overview.html [hunk 15]
```diff
@@ -1111,7 +1114,158 @@ <h3 id="referrer-policies"><span class="secno">3.4 </span>Referrer policies</h3>
...
+<ul>
+  <li><p><var title="">stream</var> is <a href="#concept-readablestream-readable" title="concept-ReadableStream-readable">readable</a>.
+  <li><p>The result of calling
+      <a href="https://streams.spec.whatwg.org/#get-readable-stream-desired-size"><code class="external" data-anolis-spec="streams">GetReadableStreamDesiredSize</code></a>(<var title="">stream</var>)
+      is positive.
+</ul><p></p>
+
+<p>A <a href="#concept-readablestream" title="concept-ReadableStream">ReadableStream</a> <var title="">stream</var> is said to be
+<dfn id="concept-readablestream-disturbed" title="concept-ReadableStream-disturbed">disturbed</dfn> if the result of calling
+<a href="https://streams.spec.whatwg.org/#is-readable-stream-disturbed"><code class="external" data-anolis-spec="streams">IsReadableStreamDisturbed</code></a>(<var title="">stream</var>) is true.
```
Comment (annevk) [inline Overview.html]: Can you add two newlines here. `<h2>` is typically preceded by three newlines in this document.

File: Overview.html [hunk 16]
```diff
@@ -1111,7 +1114,158 @@ <h3 id="referrer-policies"><span class="secno">3.4 </span>Referrer policies</h3>
...
+</ol>
+<p></p>
+
+<p>To <dfn id="concept-construct-readablestream" title="concept-construct-ReadableStream">construct</dfn> a
+<a href="#concept-readablestream" title="concept-ReadableStream">ReadableStream</a> with given <var title="">strategy</var>,
+<var title="">pull</var> action and <var title="">cancel</var> action, all of which are optional, run
+these steps:
 
+<ol>
+  <li><p>Let <var title="">init</var> be a new object.
```
Comment (annevk) [inline Overview.html]: Newline after each `<li>` here too.

File: Overview.html [hunk 17]
```diff
@@ -1111,7 +1114,158 @@ <h3 id="referrer-policies"><span class="secno">3.4 </span>Referrer policies</h3>
...
+      <var title="">init</var> as an argument. Rethrow any exceptions.
+  <li><p>Return <var title="">stream</var>.
+</ol><p></p>
+
+<p>To <dfn id="concept-construct-fixed-readablestream" title="concept-construct-fixed-ReadableStream">construct</dfn> a fixed
+<a href="#concept-readablestream" title="concept-ReadableStream">ReadableStream</a> with given <var title="">chunks</var>,
+run these steps:
+
+<ol>
+  <li><p>Let <var title="">stream</var> be the result of
```
Comment (annevk) [inline Overview.html]: And newline after each `<li>` here.

File: Overview.html [hunk 18]
```diff
@@ -1111,7 +1114,158 @@ <h3 id="referrer-policies"><span class="secno">3.4 </span>Referrer policies</h3>
...
+  <li>Return <var title="">stream</var>.
+</ol><p></p>
+
+<p>To <dfn id="concept-get-reader" title="concept-get-reader">get</dfn> a reader from a
+<a href="#concept-readablestream" title="concept-ReadableStream">ReadableStream</a> <var title="">stream</var>, run these steps:
+<ol>
+  <li><p>Let <var title="">reader</var> be the result of calling
+      <a href="https://streams.spec.whatwg.org/#acquire-readable-stream-reader"><code class="external" data-anolis-spec="streams">AcquireReadableStreamReader</code></a>(<var title="">stream</var>).
+      Rethrow any exceptions.
+  <li><p>Return <var title="">reader</var>.
```
Comment (annevk) [inline Overview.html]: And newline after each `<li>` here.

File: Overview.src.html [hunk 19]
```diff
@@ -1056,7 +1056,158 @@ <h4 id=terminology-headers>Headers</h4>
...
+
+<p>To <dfn title=concept-enqueue-ReadableStream>enqueue</dfn> a byte sequence
+(<var title>data</var>) into a <span title=concept-ReadableStream>ReadableStream</span>
+<var title>stream</var>, run the these steps:
+
+<ol>
+ <li><p>Call <code data-anolis-spec=streams>EnqueueInReadableStream</code> (<var title>stream</var>,
+     <var title>data</var>). Rethrow any exceptions.
+</ol>
+</p>
```
Comment (annevk) [inline Overview.src.html]: Please remove this `</p>`. It's wrong.

File: Overview.src.html [hunk 20]
```diff
@@ -1056,7 +1056,158 @@ <h4 id=terminology-headers>Headers</h4>
...
+
+<p>To <dfn title=concept-close-ReadableStream>close</dfn> a
+<span title=concept-ReadableStream>ReadableStream</span> <var title>stream</var>, run these steps:
+<ol>
+  <li><p>Call <code data-anolis-spec=streams>CloseReadableStream</code>
+      (<var title>stream</var>). Rethrow any exceptions.
+</ol>
+</p>
```
Comment (annevk) [inline Overview.src.html]: Please remove this `</p>`. It's wrong.

File: Overview.src.html [hunk 21]
```diff
@@ -1056,7 +1056,158 @@ <h4 id=terminology-headers>Headers</h4>
...
+      <var title>pull</var> is given.
+  <li><p>Set <var title>init</var>["cancel"] to a function that runs <var title>cancel</var> if
+      <var title>cancel</var> is given.
+  <li><p>Set <var title>init</var>["strategy"] to <var title>strategy</var> if
+      <var title>strategy</var> is given.
+  <li><p>Set <var title>init</var> be the result of calling the initial value of
+      <span title=concept-ReadableStream>ReadableStream</span> as constructor with
+      <var title>init</var> as an argument. Rethrow any exceptions.
+  <li><p>Return <var title>stream</var>.
+</ol></p>
```
Comment (annevk) [inline Overview.src.html]: Please remove this `</p>`. It's wrong.

File: Overview.src.html [hunk 22]
```diff
@@ -1056,7 +1056,158 @@ <h4 id=terminology-headers>Headers</h4>
...
+  <li><p>Let <var title>stream</var> be the result of
+      <span title=concept-construct-ReadableStream>constructing</span> a
+      <span title=concept-ReadableStream>ReadableStream</span>. Rethrow any exceptions.
+  <li><p>For each <var title>chunk</var> in <var title>chunks</var>,
+      <span title=concept-enqueue-ReadableStream>enqueue</span> <var title>chunk</var> into
+      <var title>stream</var>. Rethrow any exceptions.
+  <li><p><span title=concept-close-ReadableStream>Close</span> <var title>stream</var>. Rethrow any
+      exceptions.
+  <li>Return <var title>stream</var>.
+</ol></p>
```
Comment (annevk) [inline Overview.src.html]: Please remove this `</p>`. It's wrong.

File: Overview.src.html [hunk 23]
```diff
@@ -1056,7 +1056,158 @@ <h4 id=terminology-headers>Headers</h4>
...
+
+<p>To <dfn title=concept-get-reader>get</dfn> a reader from a
+<span title=concept-ReadableStream>ReadableStream</span> <var title>stream</var>, run these steps:
+<ol>
+  <li><p>Let <var title>reader</var> be the result of calling
+      <code data-anolis-spec=streams>AcquireReadableStreamReader</code>(<var title>stream</var>).
+      Rethrow any exceptions.
+  <li><p>Return <var title>reader</var>.
+</ol></p>
```
Comment (annevk) [inline Overview.src.html]: Please remove this `</p>`. It's wrong.

File: Overview.src.html [hunk 24]
```diff
@@ -1056,7 +1056,158 @@ <h4 id=terminology-headers>Headers</h4>
...
+
+      <li><p>When <var title>read</var> is fulfilled with a value that matches with neither of the
+          above patterns, reject <var title>promise</var> with a <code title>TypeError</code>.
+
+      <li><p>When <var title>read</var> is rejected with an error, reject <var title>promise</var>
+          with that error.
+  </ul>
+
+  <li><p>Return <var title>promise</var>.
+</ol></p>
```
Comment (annevk) [inline Overview.src.html]: Please remove this `</p>`. It's wrong.

**Comment by annevk** [comment]
In general it might be better to just remove all `</p>` since I never use them.

File: Overview.src.html (hunk 13 above)
Comment (yutakahirano) [inline Overview.src.html]: Done.

File: Overview.src.html (hunk 14 above)
Comment (yutakahirano) [inline Overview.src.html]: Done.

File: Overview.html (hunk 15 above)
Comment (yutakahirano) [inline Overview.html]: Done.

File: Overview.html (hunk 16 above)
Comment (yutakahirano) [inline Overview.html]: Done.

File: Overview.html (hunk 17 above)
Comment (yutakahirano) [inline Overview.html]: Done.

File: Overview.html (hunk 18 above)
Comment (yutakahirano) [inline Overview.html]: Done.

File: Overview.src.html (hunk 19 above)
Comment (yutakahirano) [inline Overview.src.html]: Done.

File: Overview.src.html (hunk 20 above)
Comment (yutakahirano) [inline Overview.src.html]: Done.

File: Overview.src.html (hunk 21 above)
Comment (yutakahirano) [inline Overview.src.html]: Done.

File: Overview.src.html (hunk 22 above)
Comment (yutakahirano) [inline Overview.src.html]: Done.

File: Overview.src.html (hunk 23 above)
Comment (yutakahirano) [inline Overview.src.html]: Done.

File: Overview.src.html (hunk 24 above)
Comment (yutakahirano) [inline Overview.src.html]: Done.

**Comment by yutakahirano** [comment]
Thanks for the review. Removed all `</p>`.

**Comment by yutakahirano** [comment]
By the way, the change removes three `?` from Overview.html. That may be the same problem with https://github.com/whatwg/xref/pull/7. I can revert the removal manually if it's needed.

**Comment by annevk** [comment]
`Overview.src.html` changes look good. Wonder if @domenic can have a look?

I can care of the generated differences with `?` myself when merging.

File: Overview.src.html [hunk 25]
```diff
@@ -1056,6 +1056,167 @@ <h4 id=terminology-headers>Headers</h4>
 <p class=note>The details of referrer policies are discussed in Referrer Policy.
 <span data-anolis-ref>REFERRER</span>
 
+<h3>Streams</h3>
+
+<h4>ReadableStream</h4>
+
+<p>A <dfn title=concept-ReadableStream>ReadableStream</dfn> represents a stream of data as
+specified in <cite>Streams</cite>. In this section, we define common operations for ReadableStream.
```
Comment (domenic) [inline Overview.src.html]: @annevk: is `<cite>Streams</cite>` the usual way of doing this? Seems a bit out of place.

File: Overview.src.html [hunk 26]
```diff
@@ -1056,6 +1056,167 @@ <h4 id=terminology-headers>Headers</h4>
...
+<span data-anolis-ref>STREAMS</span>
+
+<p>To <dfn title=concept-enqueue-ReadableStream>enqueue</dfn> a byte sequence
+(<var title>data</var>) into a <span title=concept-ReadableStream>ReadableStream</span>
```
Comment (domenic) [inline Overview.src.html]: chunk, instead of data?

File: Overview.src.html [hunk 27]
```diff
@@ -1056,6 +1056,167 @@ <h4 id=terminology-headers>Headers</h4>
...
+ <li><p>Let <var title>bytes</var> be an empty byte sequence.
+
+ <li>
+ <p>Let <var title>read</var> be the result of calling
+ <code data-anolis-spec=streams>ReadFromReadableStreamReader</code>(<var title>reader</var>).
+
+ <ul>
+  <li><p>When <var title>read</var> is fulfilled with an object whose <code title>done</code>
+  property is false and whose <code title>value</code> property is a
+  <code title>Uint8Array</code>, append the <code title>value</code> property to
```
Comment (domenic) [inline Overview.src.html]: Is there a better way to reference Uint8Array and TypeError?

File: Overview.src.html [hunk 28]
```diff
@@ -1056,6 +1056,167 @@ <h4 id=terminology-headers>Headers</h4>
...
+ Rethrow any exception.
+</ol>
+
+<p>An <dfn title=concept-empty-ReadableStream>empty</dfn>
+<span title=concept-ReadableStream>ReadableStream</span> is the result of
+<span title=concept-construct-fixed-ReadableStream>constructing</span> a fixed
+<span title=concept-ReadableStream>ReadableStream</span> with an empty array.
+
+<p class="note no-backref">Constructing an <span title=concept-empty-ReadableStream>empty</span>
+<span title=concept-ReadableStream>ReadableStream</span> must not throw an exception.
```
Comment (domenic) [inline Overview.src.html]: I think s/must/will/ is a bit more accurate

**Comment by domenic** [comment]
Reviewed via [HTML diff](http://services.w3.org/htmldiff?doc1=https%3A%2F%2Ffetch.spec.whatwg.org%2F&doc2=https%3A%2F%2Fraw.githubusercontent.com%2Fyutakahirano%2Ffetch%2Finfrastructure%2FOverview.html#streams). LGTM with minor nits and questions about Anolis and editorial conventions that @annevk can better address.

File: Overview.src.html (hunk 26 above)
Comment (yutakahirano) [inline Overview.src.html]: Done.

File: Overview.src.html (hunk 28 above)
Comment (yutakahirano) [inline Overview.src.html]: Done.

File: Overview.src.html (hunk 25 above)
Comment (annevk) [inline Overview.src.html]: I guess I usually don't use `<cite>Standard</cite>` and instead just add "[STREAMS]" at the end of a paragraph when the paragraph refers to something foreign. We should probably mention `<cite>Streams</cite>` in the first paragraph of https://fetch.spec.whatwg.org/#infrastructure of we wanted to do that.

File: Overview.src.html (hunk 27 above)
Comment (annevk) [inline Overview.src.html]: Not really.

File: Overview.src.html (hunk 25 above)
Comment (yutakahirano) [inline Overview.src.html]: How about https://github.com/yutakahirano/fetch/commit/991030743aa41a642e6cea792f37e22c60445ade?

**Comment by annevk** [comment]
Okay, if you could rebase this that would be great, if not, I can take care of that. Also, it might be good for @wanderview to take a look before this becomes part of the specification although if he's busy we'll just merge it in and extra review can come later.

**Comment by yutakahirano** [comment]
Rebased. Should I squash them into one commit as well?

**Comment by annevk** [comment]
Yes, please.

**Comment by yutakahirano** [comment]
Done.

**Comment by annevk** [comment]
Apologies for the delay. This looks good to me. Should we include a note after `<h3 id="streams"><span class="secno">3.5 </span>Streams</h3>` that this whole section is expected to move into IDL once https://www.w3.org/Bugs/Public/show_bug.cgi?id=28988 is fixed?

**Comment by yutakahirano** [comment]
I added a note. IIRC we have not yet agreed on the final solution (https://github.com/yutakahirano/fetch-with-streams/pull/50#issuecomment-126411742, https://github.com/yutakahirano/fetch-with-streams/issues/52) so I used a weaker expression.

**Comment by annevk** [comment]
I think that per those issues it's pretty clear it needs to move into IDL, but I'm happy with a weaker note of sorts.

**Pull request closed without merging**